x-pack/plugins/security_solution/public/detection_engine/rule_management_ui/components/maintenance_window_callout/maintenance_window_callout.tsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { MaintenanceWindowStatus } from '@kbn/alerting-plugin/common';
+import { useFetchActiveMaintenanceWindows } from './use_fetch_active_maintenance_windows';
+import * as i18n from './translations';
+
+export function MaintenanceWindowCallout(): JSX.Element | null {
+  const { data } = useFetchActiveMaintenanceWindows();
+  const activeMaintenanceWindows = data || [];
+
+  if (activeMaintenanceWindows.find(({ status }) => status === MaintenanceWindowStatus.Running)) {
```

Comment:
nit -> can use `some` instead of `find`?

Reply:
Good idea! Thanks.